### PR TITLE
Simplify to check only namespace.

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

**Comment by samuelgarcia** [comment]
@alejoe91 : if you follow the link you can see that travis is able to build the doc (here an error we have to fix).

We need now to push to github. I will see tomoroow.

**Comment by alejoe91** [comment]
I'll release spikemetrics and spiketoolkit to fix it :)

**Comment by samuelgarcia** [comment]
OK

**Comment by alejoe91** [comment]
released

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,36 @@ os:
...
 python:
   - '3.6'
+sudo: enabled
+addons:
+  apt:
+    packages:
+    - mpich
+    - libmpich-dev
+    - libglib2.0-0
+    - libgl1-mesa-glx
```
Comment (alejoe91) [inline .travis.yml]: we don't need this. This is to install spikyng-circus. We only need klusta, ml_ms4_alg, and tridesclous for the examples

File: requirements.txt [hunk 2]
```diff
@@ -3,5 +3,9 @@ spiketoolkit>=0.4.0
 spikesorters>=0.1.4
 spikewidgets>=0.2.3
 spikecomparison>=0.1.0
+
 # temporary until spikesorters MdaExtractor is removed...
 requests
+
```
Comment (alejoe91) [inline requirements.txt]: update requirements:
spikeextractors>= 0.5.6
spiketoolkit>=0.4.1
spikemetrics>=0.1.2

**Comment by alejoe91** [comment]
should I merge? @samuelgarcia are the imports ok?

**Pull request merged**